### PR TITLE
Regenerate docs in master

### PR DESCRIPTION
**Pull request by asklar**
This is the first update, for what will be the 0.65 docs

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/react-native-windows-samples/pull/421)

### CODE REVIEW

**Comment by asklar** [comment]
@stmoy @vmoroz FYI
The content has already been reviewed on the code repo since these are coming from the IDLs, let me know if you have any questions/concerns.

File: docs/IReactContext-api-windows.md [hunk 1]
```diff
@@ -7,71 +7,61 @@ Kind: `interface`
...
 
-### JsiRuntime
-`readonly`  [`JsiRuntime`](JsiRuntime) `JsiRuntime`
+### JSRuntime
+`readonly`  Object `JSRuntime`
 
-Get the JSI runtime for the running React instance. It can be null if Web debugging is used.
+Gets the JavaScript runtime for the running React instance.
+It can be null if Web debugging is used.
+**Node: do not use this property directly. It is an experimental property that may be removed or changed in version 0.65.
```
Comment (chrisglein) [inline docs/IReactContext-api-windows.md:30]:
```suggestion
**Note: do not use this property directly. It is an experimental property that may be removed or changed in version 0.65.
(realize will need to be fixed in the original, but still calling out errors as I see them)
```

File: docs/ReactInstanceSettings-api-windows.md [hunk 2]
```diff
@@ -21,16 +21,17 @@ Base path used for the location of the bundle.
...
  string `DebugBundlePath`
 
-When loading from a bundle server (such as metro), this is the path that will be requested from the server.  If this is not provided the value of [`JavaScriptBundleFile`](#javascriptbundlefile) or [`JavaScriptMainModuleName`](#javascriptmainmodulename) is used.
+When loading from a bundle server (such as metro), this is the path that will be requested from the server. If this is not provided the value of [`JavaScriptBundleFile`](#javascriptbundlefile) or [`JavaScriptMainModuleName`](#javascriptmainmodulename) is used.
 
 ### DebugHost
  string `DebugHost`
 
-> **Deprecated**: This has been replaced with [`SourceBundleHost`](#sourcebundlehost) and [`SourceBundlePort`](#sourcebundleport) and will be removed in a future version.
+> **Deprecated**: This has been replaced with [`SourceBundleHost`](#sourcebundlehost) and [`SourceBundlePort`](#sourcebundleport) and will be removed in a version 0.65.
```
Comment (chrisglein) [inline docs/ReactInstanceSettings-api-windows.md:34]:
```suggestion
> **Deprecated**: This has been replaced with [`SourceBundleHost`](#sourcebundlehost) and [`SourceBundlePort`](#sourcebundleport) and will be removed in version 0.65.

```
Also... aren't these the doc updates for 0.65? Shouldn't this be removed now? Do we have tracking bugs for all of these?

**Review by chrisglein — APPROVED** [review]

File: docs/ReactInstanceSettings-api-windows.md (hunk 2 above)
Comment (asklar) [reply docs/ReactInstanceSettings-api-windows.md:34]: The content here reflects what is in the code repo master branch. The work to remove these APIs has not been done. I don't believe we have bugs tracking it but we should.

Comment (asklar) [reply docs/ReactInstanceSettings-api-windows.md:34]: Filed microsoft/react-native-windows#7486

**Pull request merged**